compiler/control/OMROptions.cpp

```diff
@@ -100,6 +100,7 @@ TR::OptionTable OMR::Options::_jitOptions[] = {
         TR::Options::setStaticNumeric, (intptrj_t)&OMR::Options::_abstractTimeToReduceInliningAggressiveness, 0, "F%d", NOT_IN_SUBSET },
    {"acceptHugeMethods",     "O\tallow processing of really large methods", SET_OPTION_BIT(TR_ProcessHugeMethods), "F" },
    {"activateCompThreadWhenHighPriReqIsBlocked", "M\tactivate another compilation thread when high priority request is blocked",  SET_OPTION_BIT(TR_ActivateCompThreadWhenHighPriReqIsBlocked), "F", NOT_IN_SUBSET},
+   {"aggressiveLoopVersionerOpts", "O\tUse more loop versioning", SET_OPTION_BIT(TR_aggressiveLoopVersionerOpts), "F" },
```

Comment:
We have several precedents for non-imperative options, and I wasn't aware of an intent to make Xjit options follow an imperative naming scheme. Isn't it reasonable to consider any boolean option foo to be implicitly equivalent to enableFoo, with the 'enable' omitted for brevity?

I have no particular attachment to the name though, so happy to change it if you wish.